health_check/contrib/celery/backends.py

```diff
@@ -1,29 +1,66 @@
+from celery.app import default_app as app
 from django.conf import settings
 
 from health_check.backends import BaseHealthCheckBackend
-from health_check.exceptions import (
-    ServiceReturnedUnexpectedResult, ServiceUnavailable
-)
-
-from .tasks import add
+from health_check.exceptions import ServiceUnavailable
 
 
 class CeleryHealthCheck(BaseHealthCheckBackend):
+    correct_ping_response = {"ok": "pong"}
```

Comment:
constants should be uppercase

Reply:
Man, I was previously instructed to change that from uppercase to lowercase. I thought myself it was wrong but didn't question it to smooth things as this check has been pending for so long. So, I'm willing to change it to uppercase but honestly it's the last time I do for that member :-)

Reply:
that's my fault, sorry @maszaa!

I'm [`wemake-python-styleguide`](https://github.com/wemake-services/wemake-python-styleguide) user and contributor, where is one [rule](https://wemake-python-stylegui.de/en/latest/pages/usage/violations/naming.html#wemake_python_styleguide.violations.naming.UpperCaseAttributeViolation) that forbids upper-case class attributes because:

> Constants with upper-case names belong on a module level.

which IMHO in most cases (excluding e.g. `enum.Enum` subclasses) is true, however, I shouldn't give you this suggestion to change it to lowercase without examining `django-health-check` coding style.